app/experimenter/experiments/tests/api/v5/test_serializers.py

```diff
@@ -0,0 +1,28 @@
+from django.test import TestCase
+
+from experimenter.experiments.api.v5.serializers import NimbusExperimentSerializer
+from experimenter.experiments.models import NimbusExperiment
+from experimenter.experiments.tests.factories import NimbusExperimentFactory
+
+
+class TestNimbusExperimentSerializer(TestCase):
+    maxDiff = None
+
+    def test_serializer_outputs_expected_schema_for_accepted(self):
+        experiment = NimbusExperimentFactory.create_with_status(
```

Comment:
You can chop `accepted` off the title, and use `create` instead of `create_with_status`, since there isn't any per-status specific logic in this serializer (yet!).